.github/workflows/build-images.yaml

```diff
@@ -66,17 +102,41 @@ jobs:
         id: add-latest-tag
         run: |
           echo "IMG_TAGS=latest ${{ env.IMG_TAGS }}" >> $GITHUB_ENV
-      - name: Run make bundle
-        run: make bundle REGISTRY=${{ env.IMG_REGISTRY_HOST }} ORG=${{ env.IMG_REGISTRY_ORG }} IMAGE_TAG=${{ github.ref_name }}
+      - name: Add release tag
+        if: ${{ github.ref_name != env.MAIN_BRANCH_NAME }}
+        id: add-branch-tag
+        run: |
+          TAG_NAME=${GITHUB_REF_NAME/\//-}
+          echo "TAG_NAME=${TAG_NAME}" >> $GITHUB_ENV
+          echo "IMG_TAGS=${TAG_NAME} ${{ env.IMG_TAGS }}" >> $GITHUB_ENV
+      - name: Set Operator version
+        id: operator-version
+        run: |
+          if [[ ${GITHUB_REF_NAME/\//-} =~ ^v[0-9]+\.[0-9]+\.[0-9]+(-.+)?$ ]]; then
+          echo "VERSION=${GITHUB_REF_NAME/\//-}" >> $GITHUB_ENV
+          else
+          echo "VERSION=${{ github.sha }}" >> $GITHUB_ENV
+          fi
       - name: Install qemu dependency
         run: |
           sudo apt-get update
           sudo apt-get install -y qemu-user-static
+      - name: Run make bundle
+        run: |
+          if [[ ${github.event_name} == "workflow_call" || ${github.event_name} == "workflow_dispatch" ]]; then
+          echo "Building bundle via workflow call"
+          make bundle REGISTRY=${{ env.IMG_REGISTRY_HOST }} ORG=${{ env.IMG_REGISTRY_ORG }} VERSION=${TAG_NAME/v/} \
+            AUTHORINO_OPERATOR_VERSION=${github.event.inputs.authorinoOperatorBundleVersion} \
+            LIMITADOR_OPERATOR_VERSION=${github.event.inputs.limitadorOperatorBundleVersion} \
+            WASM_SHIM_VERSION=${github.event.inputs.wasmShimVersion}
+          else
+          make bundle REGISTRY=${{ env.IMG_REGISTRY_HOST }} ORG=${{ env.IMG_REGISTRY_ORG }} IMAGE_TAG=${{ github.ref_name }}
```

Comment:
Is the special treatment for on_push needed? Wouldn't it also work if you supplied it with "latest" tags in `AUTHORINO_OPERATOR_VERSION`?

Reply:
Actually, I forgot to remove that conditional... but the default values in the Makefile target are set to latest.